src/constants/markets.ts

```diff
@@ -1,15 +1,27 @@
-import { Asset, Nullable, PerpetualMarket } from '@/constants/abacus';
+import { Nullable } from '@/constants/abacus';
 import { STRING_KEYS } from '@/constants/localization';
 
 export type MarketData = {
-  asset: Asset;
-  tickSizeDecimals: Nullable<number>;
-  oneDaySparkline?: number[];
-  isNew?: boolean;
+  id: string;
```

Comment:
nit: could we add a comment about the diff between id/assetId/displayId for future readers?

Reply:
Added! Ty for feedback